dev_docs/tutorials/performance/running_performance_journey_in_cloud.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: kibDevTutorialRunningPerformanceJourneyInCloud
+slug: /kibana-dev-docs/tutorial/performance/running_performance_journey_in_cloud
+title: Running Performance Journey In Cloud
+summary: Learn how to run performance journey against Cloud cluster
+date: 2023-05-04
+tags: ['kibana', 'onboarding', 'setup', 'performance', 'development', 'telemetry']
+---
+
+## Overview
+As a way to better understand user experience with Kibana in cloud, we support running performance journeys against
+Cloud deployments.
+The process takes a few steps:
+- Create a cloud deployment
```

Comment:
Is there a minimum setting (RAM/zones) for Kibana? or the default kibana setting enough?

Reply:
I think deployment size is up to each team to decide as we probably would like to test different ones and compare. We might define some defaults later.